inputs/wysiwyg.vue

```diff
@@ -736,6 +804,9 @@
         });
       });
 
+      // add hints
+      if (maxLength) tooLong = hint(editor, {color: errorColor});
```

Comment:
please use brackets. I'm not sure why you're not getting a linter error in your text editor about them 🙁

Reply:
![screen shot 2018-08-10 at 1 54 21 pm](https://user-images.githubusercontent.com/1915119/44284030-294e6900-a22e-11e8-99cb-1798ae57fbb1.png)

Reply:
Looks like this is allowed w/ current rules

![screen shot 2018-08-17 at 2 59 18 pm](https://user-images.githubusercontent.com/1915119/44284242-f193f100-a22e-11e8-9afa-6120f59735c8.png)

Reply:
huh! well ok then 🤷‍♂️